sdk/monitorexporter/azure-analytics-monitorexporter/README.md

```diff
@@ -0,0 +1,78 @@
+# Azure Anomaly Detector client library for Java
+
+TODO
+[Source code][source_code] | Package (Maven) | [API reference documentation][api_reference_doc] | [Product Documentation][product_documentation] | [Samples][sample_readme]
+
+## Getting started
+
+### Prerequisites
+
+- Java Development Kit (JDK) with version 8 or above
```

Comment:
can you copy a more current readme, that has the JDK install link and more up to date contributing section at the bottom

Reply:
Done!